packages/icons/icons.yml

```diff
@@ -10047,13 +10790,34 @@
   sizes:
     - 32
 - name: map--center
-  friendly_name: map--center
-  aliases: []
+  friendly_name: Map center
+  aliases:
+    - map
+    - travel
+    - point
+    - pin
+    - center
+    - atlas
+    - locate
+    - gps
+    - location
+    - graph
+    - directions
+    - diagram
+    - plan
+    - legend
   sizes:
     - 32
 - name: map--identify
-  friendly_name: map--identify
-  aliases: []
+  - map
```

Comment:
This line is the problem. Running the file through a yaml validator returns the following error:

```
Error : bad indentation of a sequence entry at line 10812, column 3:
      - map
      ^
```